sdk/core/azure-core-amqp/src/impl/uamqp/amqp/claim_based_security.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "private/claims_based_security_impl.hpp"
+#include "private/connection_impl.hpp"
+#include "private/management_impl.hpp"
+#include "private/session_impl.hpp"
+
+#include <azure/core/diagnostics/logger.hpp>
+#include <azure/core/internal/diagnostics/log.hpp>
+
+using namespace Azure::Core::Diagnostics::_internal;
+using namespace Azure::Core::Diagnostics;
+using namespace Azure::Core::Amqp::_internal;
```

Comment:
Spot checking the difference, and it looks like this file is identical to the one:
`sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/claim_based_security.cpp`

Except for this line 14 is above `namespace Azure { namespace Core { namespace Amqp { namespace _detail {` here, and below in the rust amqp impl version.

Should they be identical?
Can you explain why we need two copies for what you are trying to achieve?

Reply:
It should be identical - when this was written.

Assuming that I can figure out how to fix the feature branch (it's currently utterly broken), they will be different sometime today.